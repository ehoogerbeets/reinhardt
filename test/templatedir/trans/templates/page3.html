{% blocktrans with foo=page.name comment "This is the translator's comment" %}
This is the third block trans with multiple
lines in the source and a renamed variable in it
and a translator's comment which should not appear 
here in the output. Here is the page name: {{foo}} 
{% endblocktrans %}
