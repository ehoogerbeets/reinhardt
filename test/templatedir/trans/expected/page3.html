 Dies ist der dritte trans-Block mit mehreren Zeilen in der Quelle und eine umbenannte Variable drin und Kommentar des Ãœbersetzers, die in der Ausgabe nicht hier erscheinen soll. Hier ist der Name der Seite: Test page 
